Source/OpenTK/Exceptions.cs

```diff
@@ -33,7 +33,11 @@ namespace OpenTK
     /// <summary>
     /// This exception is thrown when a GraphicsContext property cannot be changed after creation.
     /// </summary>
+#if _NET_CORECLR
+    public class ContextExistsException : Exception
```

Comment:
I think you shouldn't need this since you have a definition for ApplicationException (the one I commented on below).